optimum/commands/onnxruntime/quantize.py

```diff
@@ -70,7 +69,14 @@ class ONNXRuntimmeQuantizeCommand(BaseOptimumCLICommand):
     def parse_args(parser: "ArgumentParser"):
         return parse_args_onnxruntime_quantize(parser)
 
+    def check_requirements(self):
+        if not is_onnxruntime_available():
+            raise ImportError("Onnxruntime is not installed. Please install Onnxruntime first.")
+
```

Comment:
Why is this needed? Can't we just run the code in `run()`, and if a dependency is needed an importerror will be raised anyway no?

Reply:
Both can be done that is right, I do not have a strong opinion on which way is best. 
I would say that with the method being called it is more explicit that the command writer needs to be the one checking that everything is installed at runtime. But since it's mostly for internal use as long as we know...

Reply:
I prefer not adding extra check. The error log would be more comprehensive. Check removed.